shandler merge to master

shandlermason opened the pull request.

fireph commented on Mytinerary/Info.plist:
```diff
@@ -20,6 +20,10 @@
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>This application requires location services to work</string>
```
might be nice to still have the app work without location services enabled, it just won't show your current location

fireph commented:
A quick summary of all the things this pull request it doing would be great as well as a screenshot showing off the new map view :D

shandlermason commented:
<img width="372" alt="Screen Shot 2019-07-26 at 1 36 58 PM" src="https://user-images.githubusercontent.com/46688470/61980289-26660780-afab-11e9-98b4-ad8d4df059cd.png">
<img width="372" alt="Screen Shot 2019-07-26 at 1 43 13 PM" src="https://user-images.githubusercontent.com/46688470/61980340-56150f80-afab-11e9-965b-083ba0296493.png">

Implemented the map view and shows current user location on phone. Implement user persisted when logging into profile. Implemented Google API to allow for query search of events addresses.
